### DISCUSSION TITLE
OR 1.4.4 Multiplayer Server

### DISCUSSION THREAD

**Discussion by perpetualKid** [perpetualKid/FreeTrainSimulator#98] [General]
A long standing ask in the communities, this release of ORTS-MG includes a simple standalone server for multiplayer games. 
The MultiPlayer server can be started right out from Menu under Tools:

![Start Server](https://raw.githubusercontent.com/wiki/perpetualKid/ORTS-MG/media/Server_Menu.png)
 
Alternatively, use a commandline and start `.\multiplayerserver.exe` (Windows), or use `dotnet .\multiplayerserver.dll` (Linux, MacOS, Windows as well). A single parameter can be passed to configure a TCP Port, else the standard Port 30000 will be used. 
When started from the Menu, the configured port number will be used (50000 in the sample above).

Running under Window you may see a Firewall Warning at first start which you should carefully read before acknowledging.

Once started, the server window will print all communication to the output window: 

![Multiplayer Server Running](https://raw.githubusercontent.com/wiki/perpetualKid/ORTS-MG/media/Multiplayer_Window.png) 

All players will join the game as Multiplayer Client only, and the first client to connect will be appointed Dispatcher. If the current Dispatcher leaves the game, the server will appoint another player as new Dispatcher.

As always, if any issues, drop a note in the [Discussions board](https://github.com/perpetualKid/ORTS-MG/discussions) or the [Gitter channel](https://gitter.im/ORTS-MG/community).

<hr /><em>This discussion was created from the release <a href='https://github.com/perpetualKid/ORTS-MG/releases/tag/v1.4.4'>OR 1.4.4 Multiplayer Server</a>.</em>

**Comment by Olomoucak22** [comment]
Hi, would it be possible to add a timestamp so I can see on the server who connected when ? 
Time/Date



 Thank you in advance
